torchinductor/decomposition.py

```diff
@@ -22,16 +22,19 @@
         aten._adaptive_avg_pool2d_backward,
         aten.addcmul,
         aten.avg_pool2d_backward,
+        aten.clamp,
         aten.clamp_max,
         aten.clamp_min,
         aten.cudnn_batch_norm,
         aten.cudnn_batch_norm_backward,
         aten.detach,
+        aten.elu,
         aten.elu_backward,
         aten._embedding_bag,
         aten.embedding_dense_backward,
         aten.expand_as,
         aten._fused_moving_avg_obs_fq_helper,
+        aten.gelu,
```

Comment:
can you spot-check something like hf_bert to make sure that gelu decomp from pytorch is fine?